ReactiveCocoa/CocoaTarget.swift

```diff
@@ -3,14 +3,29 @@ import ReactiveSwift
 import enum Result.NoError
 
 /// A target that accepts action messages.
-internal final class CocoaTarget: NSObject {
-	let observer: Observer<(), NoError>
-
-	init(_ observer: Observer<(), NoError>) {
+internal final class CocoaTarget<Value>: NSObject {
+	private let observer: Observer<Value, NoError>
+	private let transform: (Any?) -> Value
+	
+	internal init(_ observer: Observer<Value, NoError>, transform: @escaping (Any?) -> Value) {
 		self.observer = observer
+		self.transform = transform
```

Comment:
What take a `transform`? Wouldn't this always be the same implementation to cast to `Value`?